server/tso/tso.go

```diff
@@ -67,7 +66,7 @@ func (t *timestampOracle) getTimestampPath() string {
 func (t *timestampOracle) loadTimestamp() (time.Time, error) {
 	data, err := etcdutil.GetValue(t.client, t.getTimestampPath())
 	if err != nil {
-		return typeutil.ZeroTime, err
+		return typeutil.ZeroTime, errs.ErrEtcdKVGet.Wrap(err).FastGenWithCause()
```

Comment:
this error has been handled in the lower level, just return it

Reply:
I think the low-level error has been used in the previous log, so its lifetime is over. This new error should be the new one to expose to outside?

Reply:
The low-level error is already wrapped in ErrEtcdKVGet, it is unnecessary to wrap it again

Reply:
Sorry I misunderstood you. Got it!